flair/models/biomedical_entity_linking.py

```diff
@@ -482,6 +482,29 @@ def search(self, entity_mentions: List[str], top_k: int) -> List[List[Tuple[str,
         :result: list of tuples in the form: (entity / concept name, concept ids, similarity score).
         """
 
+    def build_candidate(self, candidate: Tuple[str, str, float]) -> EntityLinkingCandidate:
+        """Get nice container with all info about entity linking candidate"""
+
+        concept_name = candidate[0]
+        concept_id = candidate[1]
+        score = candidate[2]
+        database_name = self.dictionary.database_name
```

Comment:
Access "self.dictionary" is kind of a hidden assumption about the attributes of the implementation. Maybe provide the dictionary via an parameter instead?